dashboard/app/jobs/concerns/ai_rubric_config.rb

```diff
@@ -6,8 +6,7 @@ class AiRubricConfig
   #
   # Basic validation of the new AI config is done by UI tests, or can be done locally
   # by running `AiRubricConfig.validate_ai_config` from the rails console.
-  S3_AI_RELEASE_PATH = 'teaching_assistant/releases/2024-04-30-confidence-autogen-turbo/'.freeze
-
+  S3_AI_RELEASE_PATH = 'teaching_assistant/experiments/claude-test-07-2024/'.freeze
```

Comment:
What things would break if we were to split this out into a subsequent PR? I'm concerned that if we need to roll back, it will be very difficult to undo all of these changes. the goal should be two distinct steps:
1. add functionality for claude without breaking openai
2. switch to a release which uses claude

Reply:
This is an adhoc only change.